app/graphql/types/analytics/gross_revenues/object.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Types
+  module Analytics
+    module GrossRevenues
+      class Object < Types::BaseObject
+        graphql_name 'GrossRevenue'
+
+        field :amount_cents, GraphQL::Types::BigInt, null: true
+        field :currency, Types::CurrencyEnum, null: true
+        field :month, GraphQL::Types::ISO8601DateTime, null: false
```

Comment:
I'm curious about what will be stored in `month`.

Will that be first day of the month in ISO format like `2023-11-01T00:00:00+0000`?

Reply:
Yes, that's what the DB query returns: 

```
[{"month"=>"2023-11-01T00:00:00.000Z", "amount_cents"=>11016700.0, "currency"=>"EUR"}]
```